### PR TITLE
22w16b entity

### LINKED ISSUE #3110: WardenBrain#tick is more like #updateActivities
I saw something similar with other new entities like tadpoles. 
Either way, the method is not at all how the brain is ticked (it's ticked earlier according to WardenEntity#mobTick)

### PR DESCRIPTION
**Pull request by apple502j**
Entity tracked position etc.

Resolves #3110

### CODE REVIEW

File: mappings/net/minecraft/entity/TrackedPosition.mapping [hunk 1]
```diff
@@ -0,0 +1,21 @@
+CLASS net/minecraft/class_7422 net/minecraft/entity/TrackedPosition
+	FIELD field_39015 COORDINATE_SCALE D
+	FIELD field_39016 pos Lnet/minecraft/class_243;
+	METHOD method_43487 toPacketValue (D)J
+		ARG 0 value
+	METHOD method_43488 toRealValue (J)D
```
Comment (liach) [inline mappings/net/minecraft/entity/TrackedPosition.mapping]: Maybe `pack` and `unpack`, like for light data converted to long?

File: mappings/net/minecraft/entity/TrackedPosition.mapping [hunk 2]
```diff
@@ -0,0 +1,21 @@
+CLASS net/minecraft/class_7422 net/minecraft/entity/TrackedPosition
```
Comment (Shnupbups) [inline mappings/net/minecraft/entity/TrackedPosition.mapping:1]: Why not `TrackedPos` to match `BlockPos` and `GlobalPos`?

**Review by Shnupbups — APPROVED** [review]

File: mappings/net/minecraft/entity/TrackedPosition.mapping (hunk 2 above)
Comment (apple502j) [reply mappings/net/minecraft/entity/TrackedPosition.mapping:1]: Hmm, maybe.

Comment (apple502j) [reply mappings/net/minecraft/entity/TrackedPosition.mapping:1]: The stuff has always been called "tracked position" in yarn, though. Do we rename it?

Comment (frqnny) [reply mappings/net/minecraft/entity/TrackedPosition.mapping:1]: I am on phone, but if it is in double and not int then marking it as Position would also mark this difference between BlockPos and this new class.

Comment (apple502j) [reply mappings/net/minecraft/entity/TrackedPosition.mapping:1]: You can't use this as `Vec3d`/`Vec3i`/whatever anyway, this is just a wrapper with network syncing utils.

**Review by enbrain — APPROVED** [review]

**Review by liach — APPROVED** [review]

**Pull request merged**